test/controllers/base_api_controller_test.rb

```diff
@@ -6,28 +6,6 @@ def setup
     @controller = Api::V1::BaseApiController.new
   end
 
-  test "should respond to json" do
-    assert_equal [:json], @controller.mimes_for_respond_to.keys
-  end
-
-  test "should filter parameters" do
-    authenticate_with_token
-    get :about, foo: 'bar', format: :json
-    assert_equal ['foo'], @controller.send(:get_params).keys
-  end
-
-  test "should return enabled archivers, name and version" do
-    authenticate_with_token
-    get :about, format: :json
-    assert_response :success
-    response = JSON.parse(@response.body)
-    assert_equal ['data', 'type'], response.keys.sort
-    assert_equal ['archivers', 'name', 'version'], response['data'].keys.sort
-    assert_equal 'about', response['type']
-    assert_equal 'Keep', response['data']['name']
-    assert_equal VERSION, response['data']['version']
-  end
-
```

Comment:
@daniel-shuy why did you remove these tests? They were not affected by your changes, they are still passing.
Could you add them back, please?

Reply:
I've moved them to the RSpec Rails test (see https://github.com/meedan/pender/pull/233/files#diff-67cf9b88436474d7327263c179301d48f2f6cba0218053690f30d5b3ef95899d)